packages/shared/lib/core/profile-manager/api/recoverAccounts.ts

```diff
@@ -1,8 +1,16 @@
 import { get } from 'svelte/store'
-import { profileManager } from '../stores'
+
+import { AccountSyncOptions } from '@iota/wallet'
```

Comment:
To mitigate side-effects use the following:
```suggestion
import type { AccountSyncOptions } from '@iota/wallet'
```